test/e2e/overlays.rendering.test.ts

```diff
@@ -36,21 +36,23 @@ class ImageSnapshotThresholds extends MultiBrowserImageSnapshotThresholds {
         'overlays.start.flow.task.gateway',
         {
           linux: 0.001, // 0.09368089665046096%
-          windows: 0.0003, // 0.025623788967854555%
+          windows: 0.003, // 0.24558800934610941%
```

Comment:
I didn't change anything for Chromium on windows. So it's weird to have to change it 🤔

Reply:
Can you verify if you have the same issue on the **master** branch ? 
I updated the version of jest-image-snapshot this morning.

Reply:
yes checking on master - I had like over 20 failures on local :(

Reply:
I confirm that it is not passing on master as well
![image](https://user-images.githubusercontent.com/45601541/116894270-25f3a980-ac32-11eb-8d43-be4e2da4161e.png)

Reply:
So, I think we should fix that problem in another PR.

Reply:
Agree that this should be done in a dedicated PR

Reply:
I reverted all the **Tresholds** commits.

Reply:
I have tested on windows 10 pro locally and don't get any e2e test errors with the original thresholds. This seems an environment issue.